### PR TITLE
PENV-252: fix non atomic tests

### PR DESCRIPTION
**Pull request by skudasov**
Isolated tests using setup db, fixed other tests to find particular transactions/notifications in order to run in parallel or with the count.

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ jobs:
       if: github.event_name == 'schedule'
       run: |
         echo "::set-env name=TEST_ARGS::-count 300 -race -p 10 -failfast -timeout 240m"
+    - name: set default pr params
```
Comment (megge-dream) [inline .github/workflows/pr.yml:34]: Don't this override previous step if `github.event_name == 'schedule' `?

Comment (skudasov) [reply .github/workflows/pr.yml]: Thanks! fixed.

File: Makefile [hunk 2]
```diff
@@ -75,10 +75,12 @@ config: ## generate configs
 
 ci_test: ## run tests with coverage
 	go test -json -v -count 10 -timeout 20m --coverprofile=coverage.txt --covermode=atomic ./... | tee ci_test_with_coverage.json
+	go test -json -v -count 10 -timeout 20m -tags="sequential" --coverprofile=coverage.txt --covermode=atomic ./... | tee ci_test_with_coverage.json
```
Comment (pavel-moskovkin) [inline Makefile:78]: this won't start if error occurs while executing previous line. Maybe something like this?- https://github.com/insolar/go-autotests/blob/master/Makefile#L45

Comment (pavel-moskovkin) [reply Makefile:78]: and one more thing: when running `go test -tags "mytag"` also triggers to start tests without tag in addition to tests with tag. So we get two similar runs here.

Comment (skudasov) [reply Makefile]: In order to reduce clutter, I've just isolated tests explicitly using db setup. Go doesn't have a good tag system, neither regex in -run is suitable for the task, so it's better to clearly statement about test state isolation like

cleaner := InitTestDB()
	defer cleaner()

and then run it the same way in count or parallel.

**Pull request merged**